examples/sphDiagramGeneration/sph-diagram-generation.cpp

```diff
@@ -66,57 +66,115 @@ int main(int argc, char *argv[]) {
   autopas::DirectSum<autopas::sph::SPHParticle, autopas::FullParticleCell<autopas::sph::SPHParticle>> dirCont(
       boxMin, boxMax, cutoff);
 
+  autopas::sph::SPHCalcDensityFunctor densfunc;
+
+  autopas::sph::SPHCalcHydroForceFunctor hydrofunc;
+
   int numParticles = 16;
   int numIterations = 100000;
   int whichContainer = 0;
-  // int whichFunctor = 0;
-  if (argc == 5 or argc == 4) {
+  int whichFunctor = 0;
+  double skin = 0.;
+  int rebuildFrequency = 10;
+  if (argc == 7) {
+    numParticles = atoi(argv[1]);
+    numIterations = atoi(argv[2]);
+    whichContainer = atoi(argv[3]);
+    whichFunctor = atoi(argv[4]);
+    skin = atof(argv[5]);
+    rebuildFrequency = atof(argv[6]);
+  } else if (argc == 5) {
+    numParticles = atoi(argv[1]);
+    numIterations = atoi(argv[2]);
+    whichContainer = atoi(argv[3]);
+    whichFunctor = atoi(argv[4]);
+  } else if (argc == 4) {
     numParticles = atoi(argv[1]);
     numIterations = atoi(argv[2]);
     whichContainer = atoi(argv[3]);
-    //  whichFunctor = atoi(argv[4]);
   } else {
```

Comment:
a help text on how to use this would be useful but I don't know how user-friendly you want this example to be ;)

Reply:
done